### ISSUE TITLE
[Concept Entry] Sklearn nearest-neighbours

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#4895]
### Reason/inspiration (optional)

We would like a new entry on the `Nearest Neighbors` concept under Sklearn. The entry should go in a new file under `docs/content/sklearn/concepts/nearest-neighbors/nearest-neighbors.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept (if applicable)
- An `Example` section that provides an example demonstrating the concept in use (if applicable)
- A `Codebyte Example` section that provides a codebyte example demonstrating the concept in use (if applicable)

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by igarousi** [comment]
Hey @codecademy-docs! Can I take up this issue?

**Comment by NeuralNoble** [comment]
I came across the issue requesting a new entry on the Nearest Neighbours concept under sklearn. I would like to contribute to this by writing the documentation entry. Below is a brief outline of how I plan to structure the entry:

Proposed Structure
### Introduction to the Concept:
- A brief explanation of what Nearest Neighbours is and its importance.

### Syntax Section:
- Detailed syntax for using Nearest Neighbours in sklearn.

### Example Section:
- A practical example demonstrating how to use Nearest Neighbours.

### Codebyte Example Section:
- A concise code example that can be embedded in the documentation.

I will ensure to follow the concept entry template, content standards, and markdown style guide as specified.

Could you please assign this issue to me?

**Comment by mamtawardhani** [comment]
Hey @igarousi You're assigned!🎉 You can look through the [Contribution Guide](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md) to know more about contributing to Codecademy Docs. Check out the [Content Standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [Style Guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md), they’ll help you with creating your entry.

I see this is your first time contributing to Codecademy Docs. How did you find out about us?

**Comment by mamtawardhani** [comment]
@NeuralNoble another issue has been assigned to you 😃

**Comment by NeuralNoble** [comment]
can i be assigned this please as i have worked with Sklearn nearest-neighbours throughout projects? you can unassign that issue to me . please

**Comment by mamtawardhani** [comment]
> can i be assigned this please as i have worked with Sklearn nearest-neighbours throughout projects? you can unassign that issue to me . please

I'm really sorry @NeuralNoble, since the issue assignment is based on the first come first serve basis. However, I suppose there are a few more sklearn issues open, you can check and let me know 😃

**Comment by igarousi** [comment]
@mamtawardhani Great, thank you! I learned about this contributing opportunity through Codecademy courses.

**Comment by igarousi** [comment]
@mamtawardhani, just posting an update: I'm still working on this issue. I will push the changes by July 20.

**Comment by mamtawardhani** [comment]
> can i be assigned this please as i have worked with Sklearn nearest-neighbours throughout projects? you can unassign that issue to me . please

Hey @NeuralNoble are you still interested in taking up this issue?

**Comment by NeuralNoble** [comment]
yes

**Comment by mamtawardhani** [comment]
> yes

Hey @NeuralNoble You’re assigned 🎉 In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide). After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.

Is this your first contribution to Codecademy Docs? If so, we’re curious to know how you found out about contributing to Docs.

**Comment by mamtawardhani** [comment]
Unassigning author due to inactivity
